euth_wagtail/static/scss/components/_hero-unit.scss

```diff
@@ -1,37 +1,71 @@
-.hero-unit {
-    position: relative;
+.euth-hero-unit {
+    background-size: cover;
     text-align: center;
+    min-height: 400px;
+    position: relative;
+    @include rem(padding, 40px 0);
 
-    &, .hero-unit-image {
-        min-height: 573px;
-    }
-
-    .hero-unit-image {
-        background-repeat: no-repeat;
-        background-position: center center;
-        background-size: cover;
-    }
+    // centering the container vertically and horizontally
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-align: center;
```

Comment:
Check our browser requirements, I think they are quite high therefor we don't need to use legacy prefixes